app/routes/bdb/components/BdbDetail.js

```diff
@@ -364,7 +365,7 @@ export default class BdbDetail extends Component<Props, State> {
           <div className={styles.files}>
             <h3>Filer</h3>
             <ul>
-              {!company.files || company.length === 0 ? (
+              {!company.files || company.files.length === 0 ? (
```

Comment:
<3

Reply:
Haha :grinning: 

The updated flow version told me to do this, and it sounds reasonable too. :smirk: